### DISCUSSION TITLE
APIs with technical preview, tentatively, when these will be available for production.

### DISCUSSION THREAD

**Discussion by saurabhbechtel** [iTwin/itwinjs-core#5014] [Q&A]
Hi, 
My team is working on synchronizing the iModel. It has bigger files to synchronize and using Bentley Azure blob URL these source files being added to manifest connections.  

**Date - 02 Feb, 2023.**
There is one warning we see, saying - *This operation is a Technical Preview and is available for testing purposes only. Do not use in production.*

I want to know are we good to go with these APIs if we are expected to move this to production by the end of March 2023.
Like what is the tentative time to use these APIs in production.

Thanks in advance.
Saurabh

**Comment by pmconne** [comment]
Hi @saurabhbechtel.

Would you mind linking directly to the page containing that warning?

**Reply by saurabhbechtel** [reply]
Hi Pmconne, 

Please find the link - https://developer.bentley.com/apis/synchronization/

Please look into this para - 
![image](https://user-images.githubusercontent.com/110451195/217504404-de4740f2-c507-41d5-a49f-04c35b0d0243.png)

**Comment by pmconne** [comment]
Thank you @saurabhbechtel 
@timlawrence-bentley is there a planned date for this feature to exit technical preview?

**Reply by timlawrence-bentley** [reply]
Hi, 
I'm not the PM for the synchronization service (connectors themselves are separate from the service), but we are moving a few of our APIs from tech preview in the next while. For synchronization, the current estimate is early March. If you need more details, please ask Laisvune.
Thanks,

Tim
